client/src/app/components/search/search.component.ts

```diff
@@ -110,4 +107,14 @@ export class SearchComponent implements OnInit {
             width: '100%'
         }
     }
+    private getTagParam(): string{
+        let tagParam = '';
+        this.filterForm.value.tags.forEach(tag => {
+            tagParam += tag + ',';
+        });
+        if(tagParam.length > 0){
+            tagParam = tagParam.substr(0, tagParam.length-1);
+        }
+        return tagParam;
+    }
 }
```

Comment:
You can just use `.join()` for converting array to string :)

Reply:
Changed